test/format/format_inline.h

```diff
@@ -281,10 +281,22 @@ static inline void
 wt_wrap_open_cursor(WT_SESSION *session, const char *uri, const char *config, WT_CURSOR **cursorp)
 {
     WT_DECL_RET;
+    bool is_checkpoint;
 
-    /* WT_SESSION.open_cursor can return EBUSY if concurrent with a metadata operation, retry. */
-    while ((ret = session->open_cursor(session, uri, NULL, config, cursorp)) == EBUSY)
+    is_checkpoint = config != NULL && WT_PREFIX_MATCH(config, "checkpoint");
+
+retry:
+    ret = session->open_cursor(session, uri, NULL, config, cursorp);
+    /*
+     * WT_SESSION.open_cursor can return:
+     * - EBUSY if concurrent with a metadata operation.
+     * - ENOENT when trying to open a checkpoint cursor.
```

Comment:
Can we add some context here explaining why `ENOENT` on a checkpoint cursor should be retried instead of aborting the tests?